### PR TITLE
Branch email switcher

### PR DESCRIPTION
**Pull request by mxsnyder**
This creates the branch email switcher, which is pasted into the bcpl_askalibrarian form. I also added a polyfill since `Array.prototype.find()` doesn't exist in MSIE.

### CODE REVIEW

File: js/one-offs/branchEmailSwitcher.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+var bcpl = bcpl || {};
+
+bcpl.branchEmailSwitcher = (function branchEmailSwitcher($) {
+	var branchEmails = [
```
Comment (martypowell) [inline js/one-offs/branchEmailSwitcher.js]: This should probably use branch-amenities.json

File: js/utility/readme.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Utilities folder
+
+Items in this folder should not have any external dependencies beyond the libraries in the **vendor** folder.
```
Comment (martypowell) [inline js/utility/readme.md]: Nice

File: js/one-offs/branchEmailSwitcher.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+var bcpl = bcpl || {};
```
Comment (martypowell) [inline js/one-offs/branchEmailSwitcher.js]: Is there a reason we aren't using es6?

File: js/one-offs/branchEmailSwitcher.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+				return branchEmailItem.name.toLowerCase() === searchTerm.toLowerCase();
+			}
+
+			return false;
+		});
+
+		return foundEmail || '';
+	};
+
+	$(document).on('change', '#whichBranch', function whichBranchChangeEvent(changeEvent) {
```
Comment (martypowell) [inline js/one-offs/branchEmailSwitcher.js]: just extract change method into it's own function.

**Comment by mxsnyder** [comment]
Ready for another look.

**Pull request merged**